odoo/models.py

```diff
@@ -1595,7 +1595,7 @@ def _read_group_fill_results(self, domain, groupby, remaining_groupbys,
                 result[key] = line
 
         # add folding information if present
-        if field.relational and groups._fold_name in groups._fields:
+        if field.relational and groups._fold_name in groups._fields and not self._context.get('unfold_all', False):
```

Comment:
@xmo-odoo  correct, but it does not matter, because we should find a way to only load the current column, not all column at once.  So I expect that this will not be necessary